src/app/static/src/app/components/projects/ProjectHistory.vue

```diff
@@ -297,7 +297,7 @@
         renameProjectNote: string;
         projectNoteToEdit: number | null;
         editedProjectNote: string;
-        openVersion: boolean
+        openVersion: Record<number, boolean>
```

Comment:
Why did you change this ?

Reply:
so i realised when you have multiple projects, they sahre this openVersion prop hence when you expand one projects versions, they all expand, so i need to change it to an object of booleans that store whether each of them are open or not